core/tests/test_awards.py

```diff
@@ -0,0 +1,158 @@
+# Standard library imports
+import pytest
+
+# Django and third-party imports
+from django.contrib.auth import get_user_model
+from faker import Faker
+
+# Local application imports
+from core.models import Community, Post, PostAward
+
+User = get_user_model()
+fake = Faker()
+
+
+@pytest.mark.django_db()
+def test_post_award_level_1() -> None:
```

Comment:
You could use already defined fixtures here. Please look at the other tests and conftest.py file. 
Problem is not using fixtures makes those tests hard to maintain, and you have a lot of duplicated code. 
Let me know when done so I can review tests. Thank you.

Reply:
Done